spec/spec_helper.rb

```diff
@@ -182,3 +182,6 @@ def get_stderr(&block)
   end
   tmp.string
 end
+
+# TODO: We'll move this into a better place
```

Comment:
We decided we want this to be a test-only helper; we'll probably want to put this into  `spec/support/lib/module_validator.rb` or similar - since it will be autoloaded by default:

https://github.com/rapid7/metasploit-framework/blob/4ed5c59cf2bb04329bf2e0b4888b70f55bac1bdc/spec/spec_helper.rb#L46-L53